### PR TITLE
feat: add input component

### PR DESCRIPTION
**Pull request by ZajoncM**

### CODE REVIEW

**Comment by Matgorn** [comment]
Update:
- refactor: handle all states and sizes within one input component
- style: update styles after changes global styles in `main` branch
- style: update styles after changes in [Figma](https://www.figma.com/file/aAZNLti1x7RHcKO2aaVdyh/Selleo-Design-System?node-id=1473%3A3352&t=MC6atK6e1XNZEjLS-4)

@k1eu @michalgren

**Review by k1eu — APPROVED** [review]

**Review by k1eu — CHANGES_REQUESTED** [review]
When both icons are present the placeholder is behind the icons

**Comment by Matgorn** [comment]
> When both icons are present the placeholder is behind the icons

Fixed 👍

File: packages/selleo-tailwind/tailwind.config.cjs [hunk 1]
```diff
@@ -32,6 +32,7 @@ module.exports = {
       7: "96px",
       8: "144px",
       9: "192px",
+      48: "48px",
```
Comment (k1eu) [inline packages/selleo-tailwind/tailwind.config.cjs:35]: weird size, can we try to stick to the extisting spacing or try to adjust design for it?

**Review by k1eu — CHANGES_REQUESTED** [review]

**Comment by Matgorn** [comment]
Update:
- refactor: use a different approach without using position absolute, just flexbox instead
- chore: remove the 48px spacing from Tailwind config and usage
@k1eu @michalgren 

Closes #40

**Review by k1eu — APPROVED** [review]

**Pull request merged**